### ISSUE TITLE
Error recordfinder PostgreSQL

### ISSUE THREAD

**Issue by TrenchTown** [octobercms/october#5667]
I use as a database PostgreSQL. Created a field type recordfinder, when trying to get a list of products, I get an error: 

"SQLSTATE[22P02]: Invalid text representation: 7 ERROR: invalid input syntax for type integer: "" CONTEXT: unnamed portal parameter $1 = '' (SQL: select * from "ufa_nutrition_products" where "ufa_nutrition_products"."id" = limit 1)" on line 760 of C:\OSPanel\domains\ufa.local\vendor\laravel\framework\src\Illuminate\Database\Connection.php

Field:
```
menu:
    type: repeater
    prompt: Add Product
    form:
        fields:
            product:
                label: Продукт
                type: recordfinder
                list: ~/plugins/ufa/nutrition/models/product/columns.yaml
                recordsPerPage: 40
                prompt: Select product
                nameFrom: name
                descriptionFrom: slug
                useRelation: false
                modelClass: Ufa\Nutrition\Models\Product
                span: full
```
Schema:
```
Schema::create('ufa_nutrition_products', function (Blueprint $table) {
            $table->engine = 'InnoDB';
            $table->increments('id');
            $table->string('name');
});
```

October v 3.1.19

**Comment by TrenchTown** [comment]
This problem manifests itself only in the repeater field. Tell me how to fix it ?

**Comment by daftspunk** [comment]
This is fixed in v3.1.20

Thanks
